scenes/login_dialog/login_dialog.gd

```diff
@@ -0,0 +1,25 @@
+extends Control
+
+@onready var nAnimationPlayer = $AnimationPlayer
+@onready var nLayer = $CanvasLayer/Layer
+@onready var nBox = find_child("Box")
```

Comment:
Make sure to follow the gdscript naming guide. Variables are snake_case, no prefixes. So animation_player, layer and box instead of nAnimationPlayer, nLayer and nBox.

Reply:
We're following the GDScript style guide, which means no prefixes.